src/lib/swapper/swappers/CowSwapper/cowGetTradeTxs/cowGetTradeTxs.test.ts

```diff
@@ -22,15 +19,10 @@ describe('cowGetTradeTxs', () => {
     jest.resetAllMocks()
   })
 
-  it('should call cowService with correct parameters and return an empty string if the order is not fulfilled', async () => {
-    const deps: CowSwapperDeps = {
-      apiUrl: 'https://api.cow.fi/mainnet/api',
-      adapter: {} as unknown as ethereum.ChainAdapter,
-      web3: {} as Web3,
-    }
-
-    const input: TradeResult = {
+  it('should call cowService with correct parameters and return an empty string if the order is not fulfilled for ETH', async () => {
+    const input: CowTradeResult = {
       tradeId: 'tradeId1112345',
+      sellAssetChainId: 'eip155:1',
```

Comment:
Non-blocking, can replace occurrences of `eip155:1` and `eip155:100` with `ethChainId` and `gnosisChainId` from caip

Reply:
Correct, should've been `ethereumChainId` for consistency

Reply:
https://github.com/shapeshift/web/pull/4595/commits/0cc0dddcddf5cfa71691efc9782e74a74d34de92